.devcontainer/welcome-message.txt

```diff
@@ -0,0 +1,10 @@
+👋 Welcome to "Thanos" in GitHub Codespaces!
+
+🛠️  Your environment is fully setup with all the required software.
+
+🚀 All the components of Thanos are spinning up with the "quickstart" script, navigate over to the "Ports" tab to access it.
+
+ℹ️  Note: Change the visibility of "Prometheus instances" to "public" so that the API endpoints can be updated.
```

Comment:
Is it possible to configure these ports to be public from the start, instead of making users change it manually? Or is there some consideration I'm missing?

Reply:
If I recall correctly, VS Code even shows a pop-up suggesting the developer to make the ports public, right @samruddhikhandale? Or maybe I can confusing things? 😂

Reply:
Got it, I thought it might be due to security, in that case having this in the welcome message is sufficient. Thank you @samruddhikhandale.